Sources/MongoSwift/BSON/Document.swift

```diff
@@ -328,6 +328,12 @@ extension Document {
             }
         }
     }
+
+    /// An implementation identical to subscript(key: String), but offers the ability to choose a default value if the
+    /// key is missing.
+    public subscript(key: String, default defaultValue: @autoclosure () -> BSONValue) -> BSONValue {
```

Comment:
I just realized we should probably have a little example of usage here, like we do for subscript. Could you add one please?

Reply:
Done.

Reply:
Perfect.